FWCore/Framework/src/LuminosityBlockProcessingStatus.h

```diff
@@ -70,6 +70,8 @@ namespace edm {
     void setGlobalEndRunHolder(WaitingTaskHolder);
     void globalEndRunHolderDoneWaiting() { globalEndRunHolder_.doneWaiting(std::exception_ptr{}); }
 
+    bool setStreamStartedLumi(unsigned int iStream);
```

Comment:
I think it would (eventually, say after we've gotten feedback from Chris too on the viability of this approach) be good to add a comment here about the purpose of this function, and the meaning of the return value.

Reply:
I added a comment here.